container.go

```diff
@@ -747,11 +747,17 @@ func (c *Client) ExportContainer(opts ExportContainerOptions) error {
 
 // NoSuchContainer is the error returned when a given container does not exist.
 type NoSuchContainer struct {
-	ID string
+	ID  string
+	Err error
 }
 
 func (err *NoSuchContainer) Error() string {
-	return "No such container: " + err.ID
+	switch err.Err.(type) {
+	case error:
+		return "No such container: " + err.ID + " - " + err.Err.Error()
+	default:
+		return "No such container: " + err.ID
+	}
```

Comment:
Please use:

```
if err.Err != nil {
    return fmt.Sprintf("No such container: %s - %s", err.ID, err.Err)
}
return "No such container: " + err.ID
```

But I'm still thinking whether we should rename the error type.